packages/ui/src/services/global-zone/desktop-global-zone.service.ts

```diff
@@ -25,15 +25,22 @@ import type { IGlobalZoneService } from './global-zone.service';
 export class DesktopGlobalZoneService implements IGlobalZoneService {
     readonly visible$ = new Subject<boolean>();
     readonly componentKey$ = new Subject<string>();
```

Comment:
Make this a `BehaviorSubject` and give it an initial value would be simpler.

Reply:
get